Makefile

```diff
@@ -220,8 +224,9 @@ _build_local: source
 	@mv packaging/$(PKGNAME).spec.bak packaging/$(PKGNAME).spec
 
 build_container:
-	@echo "--- Build RPM ${PKGNAME}-${VERSION}-${RELEASE}.el$(DIST_VERSION).rpm in container ---"
-	@case "$(BUILD_CONTAINER)" in \
+	@[ "$(_CONTAINER_TOOL)" == "docker" -a "$$UID" -ne 0 ] && { echo "Container tool 'docker' must be run as root!"; exit 1; }; \
```

Comment:
Actually this is not true for macos, I don't need to run it as root. Also docker afaik can be configured for non root access.

Reply:
Maybe the implementation on MacOS is different. 9n Linux rootless mode was introduced in docker version 20.10 which is pretty recent and not available on a lot of systems.

Do you suggest to just drop the check?